File: 05 - Python/08-07-00-EjemploFuncionRecursiva-ULP.py
# Funciones Recursivas

def funcion_recursiva(numero):
    # Caso Base
    if numero == 1:
        print(numero)
    else:
        # Caso recursivo
        funcion_recursiva(numero - 1)
        print(numero)

# Llamamos a la funcion recursiva
funcion_recursiva(3)

File: 04 - JavaScript/08-07-00-EjemploFuncionRecursiva-ULP.js
// Funciones Recursivas

function funcionRecursiva(numero){
    // Caso Base
    if (numero == 1)
        console.log(numero)
    else{
        // Caso recursivo
        funcionRecursiva(numero - 1)
        console.log(numero)
    }
}

// Llamamos a la funcion recursiva
funcionRecursiva(3)